bibliography-sage.bib

```diff
@@ -5433,7 +5433,7 @@ @article{benito2012sage
   note         = {http://www.tcms.org.ge/Journals/TMJ/Volume5_2/Xpapers/tmj5_2_1.pdf},
   number       = {2},
   pages        = {3--16},
-  title        = {{Sage computations of $$\backslash$mathfrak $\{$sl$\}$ \_2 (k) $-Levi extensions}},
+  title        = {{Sage computations of $\mathfrak{sl}_2 (k)$-Levi extensions}},
```

Comment:
well, this errors for some reason. seems like double `{{sl}}` are necessary in formulas, to retain single quotes?